src/HDInsight/HDInsight/help/Restart-AzHDInsightHost.md

```diff
@@ -0,0 +1,203 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll-Help.xml
+Module Name: Az.HDInsight
+online version:
```

Comment:
please add online version

Reply:
Added

---
In reply to: [438670785](https://github.com/Azure/azure-powershell/pull/12124#discussion_r438670785) [](ancestors = 438670785)